src/sentry/api/endpoints/organization_repository_commits.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from sentry.api.base import DocSection
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.exceptions import ResourceNotFOund
+from sentry.api.serializers import serialize
+from sentry.models import Commit, Repository
+
+
+class OrganizationRepositoryCommitsEndpoint(OrganizationEndpoint):
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization, repo_id):
+        """
+        List a Repositories's Commits
```

Comment:
`Repository's` normally would pick on grammar, but this is used for public docs.